src/utils/functions.js

```diff
@@ -0,0 +1,29 @@
+const Discord = require("discord.js")
+module.exports = {
+    handleErrors,
+    checkSnowflake,
+}
+
+/**
+ * 
+ * @param {Discord.Message} msg 
+ * @param {Object} response - API error message
+ * @param {String} response.error - API error name
+ * @param {String} response.description - API error description
+ */
+async function handleErrors(msg, response) {
+    switch (response.error) {
+        case "AuthenticationError": {
+            switch (response.description) {
+                case "API key is wrong":
+                    return msg.channel.send("Error: API key has been set incorrectly.")
+            }
+        }
+        default:
+            msg.channel.send(`Error \`${response.error}\`: \`${response.description}\``)
+    }
+}
+
+function checkSnowflake(snowflake) {
+    // if (snowflake.match)
+}
```

Comment:
What are your plans for this fucntion?

Reply:
That was a plan to check if a string is a valid snowflake. Since they can be 17 characters and will increase over time, fetching whatever it is supposed to be from the Discord API is better (as it errors out if it can't find it). Something like src/commands/informatics/removewebhook.js#L22:
```js
try {
    await client.fetchWebhook(args[0], args[1])
} catch (e) {
    return message.channel.send("Invalid webhook")
}
```
That will either fetch the webhook or throw the error (which cancels the command).

Removed the function.

Reply:
Ah I see, the new method you found is better.